camp/sour1emon/XBox Controller Left Stick Y direction/src/main/java/frc/robot/commands/MotorDefaultCommand.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.MotorSubsystem;
+
+/**
+ * A command that sets the speed of a motor subsystem based on the Y direction
+ * of the left stick of an Xbox controller.
+ */
+public class MotorDefaultCommand extends Command {
+    private XboxController controller;
+    private MotorSubsystem motorSubsystem;
+
+    /**
+     * Constructs a MotorDefaultCommand.
+     * 
+     * @param motorSubsystem the motor subsystem to control
+     * @param controller     the Xbox controller to read input from
+     */
+    public MotorDefaultCommand(MotorSubsystem motorSubsystem, XboxController controller) {
+        this.motorSubsystem = motorSubsystem;
+        this.controller = controller;
+        addRequirements(motorSubsystem);
+    }
+
+    /**
+     * Executes the command by setting the speed of the motor subsystem based on the
+     * left Y direction of the controller.
+     */
+    @Override
+    public void execute() {
+        motorSubsystem.setSpeed(controller.getLeftY());
```

Comment:
We don't want to take a controller value directly into a motor. I see that in your subsystem, you have clamped the speed to the motor +/-1, which is good. 

We also need to provide a deadband. Which means that if the controller value is near 0, we stop the motor completely. Leaving a motor constantly driven at a small value will burn it out over time.

Here's an example of a deadband method and how we use it on Ted:
https://github.com/Team4682CyBears/Crescendo2024/blob/70c472825276ae1bd7ec6ff5325898e8c4da4e88/ted/src/main/java/frc/robot/RobotContainer.java#L364 

When you revise this command to take in a doubleSupplier, it would also be natural to implement and apply the the deadband before passing in the doubleSupplier to this class. The deadband is specific to the type of controller you are using, and so it belongs in RobotContainer, not in this command.